vdr/api/v1/api.go

```diff
@@ -120,12 +134,24 @@ func (a *Wrapper) CreateDID(ctx context.Context, request CreateDIDRequestObject)
 	if keyFlags != defaultKeyFlags {
 		options = options.With(keyFlags)
 	}
+	options = options.With(didsubject.NutsLegacyNamingOption{})
 
-	doc, _, err := a.VDR.NutsDocumentManager().Create(ctx, options)
+	docs, _, err := a.SubjectManager.Create(ctx, options)
```

Comment:
this also automatically creates a did:web DID?

Do we want that, or only if you use the v2 API?

Reply:
We'll add documentation on how to cope with the different APIs, the supported DID methods and v5/v6

Reply:
#3285